catalog.html

```diff
@@ -0,0 +1,217 @@
+                    <div></div><!--звездочки-->
+                </li>
+                <li class="product-card">
+                    <a class="product-card-link" href="#">
+                      <img class="product-card-image" src="#" alt="Desert Quail Inn">
+                    <a href="#">
+                      <h3 class="product-card-title">Desert Quail Inn</h3>
+                    </a>
+                    <p class="product-card-housing">Гостиница</p>
+                    <p class="product-card-price">от<span>2500</span>₽</p>
+                    <a class="product-card-button-more" href="#">Подробнее</a>
+                    <button type="button" name="to-favorites">В избранное</button>
+                    <p class="product-card-button-rating" href="#">Рейтинг:<span>6,9</span></p>
+                    <div></div><!--звездочки-->
+                </li>
+                <li class="product-card">
+                    <a class="product-card-link" href="#">
+                      <img class="product-card-image" src="#" alt="GreenTree Inn">
+                    </a>
+                    <a href="#">
+                      <h3 class="product-card-title">GreenTree Inn</h3>
+                    </a>
+                    <p class="product-card-housing">Гостиница</p>
+                    <p class="product-card-price">от 1500 ₽</p>
+                    <a class="product-card-button-more" href="#">Подробнее</a>
+                    <button type="button" name="to-favorites">В избранное</button>
+                    <p class="product-card-button-rating" href="#">Рейтинг:<span>5,0</span></p>
+                    <div></div><!--звездочки?-->
+                </li>
+            </ul>
+        </section>
+
+        <section class="pagination">
+        <ol class="pagination">
+          <li class="pagination-item">
+            <a class="pagination-link pagination-current">1</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">2</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">3</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">4</a>
+          </li>
+        </ol>
+            <span>...</span>
+            <a class="pagination-link" href="#">10</a>
+        </section>
+        <section class="newsletter">
+          <h2 class="page-footer-title">Подпишитесь на рассылку</h2>
+          <p class="page-footer-subtitle">Только полезная информация и никакого спама, честное бойскаутское!</p>
+          <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+            <label for="newsletter-email">Email</label>
+            <input class="field" placeholder="Ваш e-mail" type="email" name="newsletter-email" id="newsletter-email" required>
+            <button class="button newsletter-button" type="submit">Подписаться</button>
+          </form>
+    </main>
```

Comment:
```suggestion
          </form>
        </section>
    </main>
```
Пропущен закрывающий тег `</section>`.